brainpy/_src/math/ndarray.py

```diff
@@ -1017,12 +1022,12 @@ def pow(self, index: int):
     return self._value ** index
 
   def addr(self,
-            vec1: Union['Array', jax.Array, np.ndarray],
-            vec2: Union['Array', jax.Array, np.ndarray],
-            *,
-            beta: float = 1.0,
-            alpha: float = 1.0,
-            out: Optional[Union['Array', jax.Array, np.ndarray]] = None) -> Union[None, NoReturn]:
+           vec1: BmArray,
+           vec2: BmArray,
+           *,
+           beta: float = 1.0,
+           alpha: float = 1.0,
+           out: OptionalBmArray = None) -> Union[None, NoReturn]:
```

Comment:
As we have discussed, ``out`` should be a brainpy Array. Therefore, this typing should change to 

```
OptionalBmArray = Optional[Array]
```

Reply:
Moreover, ``Union[None, NoReturn]`` is also wrongly typed.

Reply:
OK, I will fix it